dev/components/conversationsList/component.js

```diff
@@ -15,10 +15,16 @@
     }
   });
 
-  ctrl.$inject = [];
-  function ctrl() {
+  ctrl.$inject = ['notificationsService'];
+  function ctrl(notificationsService) {
     var vm = this;
 
+    vm.unreadMessage = null;
```

Comment:
It is not a message, you need to find better name

Reply:
I used it because it was used inside left menu to display a number of unread messages. So I used it as a check/trigger.

Reply:
But name is incorrect